crm_compassion/security/crm_compassion_security.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="1">
+       <record id="crm_event_compassion_rule" model="ir.rule">
+         <field name="name" >Crm event Compassion multi-company</field>
+         <field name="model_id" ref='model_crm_event_compassion'/>
+         <field name="global" eval="True"/>
+         <field name="domain_force">['|',('company_id','=',False),('company_id','child_of',[user.company_id.id])]</field>
+       </record>
+       <record id="compassion_hold_rule" model="ir.rule">
+         <field name="name" >Compassion Hold multi-company</field>
+         <field name="model_id" ref='model_compassion_hold'/>
+         <field name="global" eval="True" />
+         <field name="domain_force">['|',('event_id.company_id','=',False),('event_id.company_id','child_of',[user.company_id.id])]</field>
```

Comment:
Hold objects are also related to children. So here you can combine the two fields and do the following rule:
```suggestion
         <field name="domain_force">['|', '|', ('event_id.company_id','=',False),('event_id.company_id','child_of',[user.company_id.id]), ('child_id.company_id.id', 'child_of', [user.company_id.id])]</field>
```